### PR TITLE
Create list blog section

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- #9

### CODE REVIEW

File: practices/practice-one/src/components/Blogs/Blogs.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+};
+
+function Blogs(props: BlogsProps) {
+  const { data } = props;
+  return (
+    <section className="list-blog">
+      <Grid col={4}>
+        {data.map((blog, index) => (
+          <section>
+            <Card {...blog} key={index} />
```
Comment (huypham-agilityio) [inline practices/practice-one/src/components/Blogs/Blogs.tsx]: dont use index as key

File: practices/practice-one/src/components/Blogs/Blogs.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import Card, { CardProps } from "components/Card/Card";
+import Grid from "components/Grid/Grid";
+
+type BlogsProps = {
+  data: CardProps[];
+};
+
+function Blogs(props: BlogsProps) {
+  const { data } = props;
```
Comment (huypham-agilityio) [inline practices/practice-one/src/components/Blogs/Blogs.tsx:11]: add empty line for readability

File: practices/practice-one/src/components/Blogs/Blogs.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <section className="list-blog">
+      <Grid col={4}>
+        {data.map((blog, index) => (
+          <section>
```
Comment (huypham-agilityio) [inline practices/practice-one/src/components/Blogs/Blogs.tsx]: Is the _section_ tag appropriate?

File: practices/practice-one/src/components/Blogs/Blogs.tsx (hunk 1 above)
Comment (DuongPham-dev) [reply practices/practice-one/src/components/Blogs/Blogs.tsx]: Right now I can't use id as key, so I use index instead during testing

**Review by huypham-agilityio — APPROVED** [review]

**Pull request merged**